math/power/powvialogarithm.go

```diff
@@ -0,0 +1,24 @@
+// powvialogarithm.go
+// description: Powers in terms of logarithms
+// details:
+// implementation of exponentiation using exponent and logarithm, without using loops  - [Powers via logarithms wiki](https://en.wikipedia.org/wiki/Exponentiation#Powers_via_logarithms)
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see powvialogarithm_test.go
+
+package power
+
+import (
+	"math"
+)
+
+func PowUseLog(a float64, b float64) float64 {
```

Comment:
Can you change the name of the function? if someone was to use this in there code to try it out they would have to use it like this:

```Go
import github.com/TheAlgorithms/Go/math/power
...

func main() {
    ...
    power.PowUseLog(a, b)
    ...
}

```

which is arguably not very reader friendly - specially if someone is trying to learn from - the main purpose of this repo 😄 

My recommendation would be that you have something like this:

```Go
import github.com/TheAlgorithms/Go/math/power
...

func main() {
    ...
    power.UsingLog(a, b)
    ...
}
```

so it would be easier to parse through - sorry this is actually quite pedantic but believe me, readability makes a huge difference when parsing through code 😄 .

```suggestion
func UsingLog(a float64, b float64) float64 {
```

Reply:
Totally agree with you. Fixed names